src/server/rdb_load.cc

```diff
@@ -228,6 +228,11 @@ string ModuleTypeName(uint64_t module_id) {
   return string{name};
 }
 
+bool RdbTypeAllowedEmpty(int type) {
+  return type == RDB_TYPE_STRING || type == RDB_TYPE_JSON || type == RDB_TYPE_SBF ||
```

Comment:
do we allow empty for this types?
I think this should be only 
constexpr uint8_t RDB_TYPE_HASH_WITH_EXPIRY = 31;
constexpr uint8_t RDB_TYPE_SET_WITH_EXPIRY = 32;
No?

Reply:
nope, we do allow. For strings: `SET foo ""` --> empty and for SBF we have a test :)

See `PrimeValue::TagAllowsEmptyValue`

Reply:
synced with Adi internally, we decided to also not log as errors `hash sets and sets` with expiry